src/gwt/src/org/rstudio/core/client/hyperlink/HelpPageShower.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * HelpPageShower.java
+ *
+ * Copyright (C) 2022 by RStudio, PBC
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+package org.rstudio.core.client.hyperlink;
+
+public interface HelpPageShower {
+   void showHelpPage(); 
```

Comment:
Nit: 'showHelpPopup()', to better match the intended usage? Or just `showHelp()`?